tools/refguide_check.py

```diff
@@ -948,9 +1132,9 @@ def main(argv):
     os.environ['SCIPY_PIL_IMAGE_VIEWER'] = 'true'
 
     module_names = list(args.module_names)
-    for name in list(module_names):
+    for name in module_names:
         if name in OTHER_MODULE_DOCS:
-            name = OTHER_MODULE_DOCS[name]
+            name = OTHER_MODULE_DOCS[name]    
```

Comment:
extra blank space at the end of the line

Reply:
Rectified